api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +688,156 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
 	}
 	
 	/**
-	 * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnTrueWhenFolderContainsFileWithGivenName() {
+		File folder = createFileAtRelativePath("folder");
+		folder.mkdir();
+		File file = createFileAtRelativePath("folder" + separator + "file");
+		try {
+			file.createNewFile();
+			assertTrue(OpenmrsUtil.folderContains(folder, "file"));
+			file.delete();
+
+		} 
+		catch (IOException e) {Assert.fail("Unable to create file");}
+		folder.delete();
```

Comment:
There is no test case(Behaviour assertions) once  IOException is thrown!

Reply:
If the exception is thrown the test will fail. Is it incorrect?

Reply:
I think that testing this Exception is out of scope . If we are able to create file the test will do it s job . If not test fail.

Reply:
Can you implement that?

Reply:
This is already implemented . Thats why i added this assert at catch. If the test go to the catch it will fail.